debian/control

```diff
@@ -193,7 +193,9 @@ Depends:
   wpasupplicant (>= 0.6.7),
   zabbix-agent2,
   ndppd,
-  miniupnpd-nftables
+  miniupnpd-nftables,
+  wpasupplicant (>= 0.6.7),
```

Comment:
Minor point: wpasupplicant is specified twice in that version of debian/control

Reply:
done